site/smart-accounts/accounts/light-account.md

```diff
@@ -3,7 +3,7 @@ outline: deep
 head:
   - - meta
     - property: og:title
-      content: Light Account Smart Contract
+      content: Light Account
```

Comment:
Are these changes good for SEO? we should confirm meta tag updates with Brady

Reply:
reverted